### PR TITLE
Add actor and action classes

### PR DESCRIPTION
**Pull request by thuanlhv**

### CODE REVIEW

File: openweather/src/main/java/action/SearchAction.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+public class SearchAction extends ActionBase {
+
+    private HomePage homePage;
+    private SearchPage searchPage;
+
+    public SearchAction() {
+        super();
+
+        homePage = getPageInstance(HomePage.class);
+        searchPage = getPageInstance(SearchPage.class);
```
Comment (thuanlhv) [inline openweather/src/main/java/action/SearchAction.java:17]: code hay qua !!!

Comment (minhhaivu) [reply openweather/src/main/java/action/SearchAction.java:17]: ờ, hay

Comment (minhhaivu) [reply openweather/src/main/java/action/SearchAction.java:17]: should rename to ActionSearch for consistency with ActionBase???

File: openweather/src/main/java/actor/SearchTester.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    public SearchAction searchAction = new SearchAction();
+
+    public static SearchTester getInstance() {
+        if (instance == null) {
+            instance = new SearchTester();
+        }
+
+        return instance;
+    }
+}
```
Comment (minhhaivu) [inline openweather/src/main/java/actor/SearchTester.java:23]: Do we need ActorBase?

**Review by minhhaivu — APPROVED** [review]

**Pull request merged**